### DISCUSSION TITLE
OnPointerPressed does not get invoked for Custom control

### DISCUSSION THREAD

**Discussion by ArtjomP** [AvaloniaUI/Avalonia#6378] [Q&A] [answered]
Hello there!
I created a custom control based on the [CustomSkiaPage sample](https://github.com/AvaloniaUI/Avalonia/blob/release/0.9.9/samples/RenderDemo/Pages/CustomSkiaPage.cs).
Acutally just added OnPointerPressed there and set a breakpoint inside. 
```cs
protected override void OnPointerPressed(PointerPressedEventArgs e)
{
 base.OnPointerPressed(e);
}
```
And the breakpoint never works.
What could be wrong?
Thank you in advance.

**Comment by kekekeks** [comment, marked as answer]
You need to implement hit-testing in your custom drawing operation. `public bool HitTest(Point p) => true;` would be enough for simple cases.

**Reply by ArtjomP** [reply]
Thank you!
Have a good time!
